03-Security-In-CICD/Readme.md

```diff
@@ -416,10 +417,22 @@ The parameters used are:
 
 [Back to top…](Readme.md#contents)
 
-### ARM Template Checker - What is covered?  
+### ARM Template Checker - Control coverage
+ 
 ARM Template checker covers Baseline controls for following services:
-App Service, Storage, SQL, CDN, Traffic Manager, Document DB, Redis Cache, and Data Lake.
-ARM Template for reference are available [here](../ARMTemplates).
+
+|FeatureName|Resource Type|
+|-----------|-------------|
+|[AppService](Feature/AppService.md)|Microsoft.Web/sites|
```

Comment:
Links are broken for features list.  
Please add controls list for services covered in ARM checker.

Reply:
removed broken links